WcaOnRails/app/models/competition_event.rb

```diff
@@ -7,6 +7,11 @@ class CompetitionEvent < ApplicationRecord
   has_many :rounds, -> { order(:number) }
   accepts_nested_attributes_for :rounds, allow_destroy: true
 
+  validates_numericality_of :event_entry_fee_lowest_denomination, greater_than_or_equal_to: 0
```

Comment:
Naming: `event_entry_fee`can be replaced with `fee` throughout the class.
- This class represents an event and we don't need to restate that
- To my language sense, this is not an entry fee.